readthedocs/core/utils/__init__.py

```diff
@@ -96,6 +96,9 @@ def prepare_build(
 
     if not version:
         version = project.versions.get(slug=LATEST)
+        if not version.active:
```

Comment:
I think we can just use this method https://github.com/rtfd/readthedocs.org/blob/4b823ad90f02d1ea6925eb1be72bb15f5b6e8c9a/readthedocs/projects/models.py#L950

Reply:
I am a little confused here.
Do we need to trigger build for the `default_branch` or `default_version`.

In case of `default_branch`, I can update the PR to use `get_default_branch()` and in case of `default_version` the PR can be updated to use `get_default_version()`.

Reply:
Default branch, yeah.

Reply:
Also, my understanding of the workflow is as follows:
1. Get latest version.
2. if it is not active, get `default_version`/`default_branch`.

Is it correct?

Reply:
If version is None, use the `get_default_branch` method. The active check we already do that more down the code I believe or before triggering it.

Reply:
@stsewd 
I still don't get it. Sorry for that.

> If version is None

What do you mean by this?
Also I have pushed some changes.